src/Eagle.ts

```diff
@@ -2445,13 +2447,23 @@ export class Eagle {
             Utils.showNotification("Error", "No Node Is Selected", "warning");
         }else{
             if (tableType === 'component'){
```

Comment:
This is more of a style-related thing than a correctness thing, but I think this sort of comparison is error-prone. It is too easy to misspell 'component'.

We have an enumerated type (Eagle.FieldType) which has two possible values (Field, and ApplicationParam). I'd be tempted to use this instead of the literal string 'component'.

You could also make the tableType parameter a Eagle.FieldType instead of a string.